.github/workflows/linux.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           # the test suite is sensitive to
           # relative dir for test file paths
           cd darshan-util/pydarshan
-          pytest
+          pytest --cov-report xml --cov=darshan
```

Comment:
may want to cover the folder with the test modules in it as well--that can help catch cases with tests that accidentally have the same name for example

Reply:
I've added `--cov=tests` to the above command. I tested it [on my fork](https://github.com/nawtrey/darshan/pull/2) and `/tests` showed up in the `codecov` GUI